src/components/SkyBrowser/PopoverSkybrowser.jsx

```diff
@@ -0,0 +1,197 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Resizable } from 're-resizable';
+import { excludeKeys } from '../../utils/helpers';
+import MaterialIcon from '../common/MaterialIcon/MaterialIcon';
+import Button from '../common/Input/Button/Button';
+import styles from './PopoverSkybrowser.scss';
+import WindowSkybrowser from './WindowSkybrowser';
+import PaneSkybrowser from './PaneSkybrowser';
+
+const findStyles = arr => arr.split(' ')
+  .map(style => styles[style] || style)
+  .join(' ');
+
+const WindowStyle = {
+  DETACHED: 'DETACHED',
+  PANE: 'PANE',
+  ATTACHED: 'ATTACHED',
+};
+
+class PopoverSkybrowser extends Component {
```

Comment:
The name of this class is a bit weird. Just flipping the order of the words helps a lot though: `SkybrowserPopover`

(It does so much more than any of the other popovers though 🤔 Maybe it deserves a more explanatory name. Although, I don't have a good suggestion..)